package.json

```diff
@@ -35,15 +36,14 @@
     "karma-chrome-launcher": "2.0.0",
     "karma-sauce-launcher": "^1.1.0",
     "nwb": "^0.15.6",
-    "preact": "^7.2.0",
+    "preact": "*",
     "semantic-release": "^6.3.2",
     "semistandard": "^9.0.0",
-    "skatejs-web-components": "5.2.1",
     "typescript": "^2.1.4",
     "typescript-formatter": "^5.0.0"
   },
   "peerDependencies": {
-    "preact": "^7.2.0"
+    "preact": "*"
```

Comment:
I know @developit mentioned that he is doing it like this because he run into some issues. 

In my experience this could be dangerous, because with this you say to the user, use whatever version you like, but skate may not work with every possible version of Preact right ? I think `^7.2.0` is right way ( any feat.patch of Preact 7 )